src/test/resources/no/nav/tilleggsstonader/sak/vedtak/boutgifter/beregning/beregning_midlertidig_overnatting.feature

```diff
@@ -119,7 +119,7 @@ Egenskap: Beregning av midlertidig overnatting
       | 11.01.2025 | 12.01.2025 | TILTAK    | NEDSATT_ARBEIDSEVNE |
       | 09.02.2025 | 14.02.2025 | TILTAK    | NEDSATT_ARBEIDSEVNE |
 
-    Så forvent følgende feilmelding: Vi støtter foreløpig ikke at utgifter krysser ulike utbetalingsperioder.
+    Så forvent følgende feilmelding: Utgiftsperioder krysser beregningsperioder
```

Comment:
For å teste hele feilmeldingen:
```suggestion
    Så forvent følgende feilmelding:
    """
    Utgiftsperioder krysser beregningsperioder

    Utgiftsperiode 09.02.2025–14.02.2025 krysser beregningsperiodene:
     - 11.01.2025–10.02.2025
     - 11.02.2025–10.03.2025

    Utgiftsperioden(e) må splittes.
    """
```

Merk at da trenger vi en ny "forvent følgende feilmelding" også:
```
    @Så("forvent følgende feilmelding:")
    fun `forvent følgende feilmelding`(forventetFeilmelding: String) {
        assertThat(feil).isNotNull
        assertThat(feil?.message).contains(forventetFeilmelding)
    }
```

Reply:
Takk. Har fikset det her: https://github.com/navikt/tilleggsstonader-sak/pull/737/commits/eacc8e3110c90e7ac63849a86d21a8c5e05a6a97